script/generateHistory.py

```diff
@@ -0,0 +1,84 @@
+import json
+import os
+from subprocess import PIPE, Popen
+
+adapterPath = 'adapter/'
+aggregatorDefaultPath = 'aggregator/default/'
+aggregatorBaobabPath = 'aggregator/baobab/'
+
+adapters = sorted(os.listdir(adapterPath))
+aggregatorsDefault = sorted(os.listdir(aggregatorDefaultPath))
+aggregatorsBaobab = sorted(os.listdir(aggregatorBaobabPath))
+
+
+def loadJsonFromPath(filePath):
+    with open(filePath) as json_file:
+        data = json.load(json_file)
+    return data
+
+
+def makeLine(words):
+    for word in words:
+        if type(word) == dict:
+            print('| [{}]({}) '.format(word['value'], word['url']), end='')
```

Comment:
Python f-strings are better way to make more readable dynamic strings.

```suggestion
            print(f"| [{word['value']}]({word['url']})")
```